lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -2487,7 +2487,11 @@ namespace Js
     void ParseableFunctionInfo::Finalize(bool isShutdown)
     {
         __super::Finalize(isShutdown);
-        this->GetUtf8SourceInfo()->StopTrackingDeferredFunction(this->GetLocalFunctionId());
+        if (this->GetFunctionInfo())
+        {
+            // (If function info was never set, then initialization didn't finish, so there's nothing to remove from the dictionary.)
+            this->GetUtf8SourceInfo()->StopTrackingDeferredFunction(this->GetLocalFunctionId());
```

Comment:
Can we add an assert in TrackDeferredFunction to always have functionInfo from ParseableFunctionInfo? LGTM